docs/components.md

```diff
@@ -13,8 +13,6 @@ flowchart TB
     carbon-estimation`"]
     Assumptions["`AssumptionsAndLimitationComponent
     assumptions-and-limitation`"]
-    HelperInfo["`HelperInfoComponent
```

Comment:
Just realised, you'll also need to remove the reference to HelperInfo later in this diagram (`CarbonEstimatorForm --> HelperInfo & Note`). With it there it creates a box and connection to it still.

Reply:
Ah thanks! I've removed the description of the component from the docs as well.